canvas.js

```diff
@@ -0,0 +1,169 @@
+let canvas = document.getElementById('canvas');
```

Comment:
the structure in this file makes a lot of sense to me, I can read it pretty well :)